tests/unit/multiclass/tree/RandomForest_unittest.cc

```diff
@@ -279,3 +282,126 @@ TEST(RandomForest,classify_non_nominal_test)
 	SG_UNREF(c);
 	SG_UNREF(eval);
 }
+
+TEST(RandomForest, test_probabilities)
+{
+	sg_rand->set_seed(1);
+
+	SGMatrix<float64_t> data(4, 14);
+	SGVector<float64_t> lab(14);
+
+	generate_nm_data(data, lab);
```

Comment:
generate_nm_data is not a good name for a function. It doesnt tell me what "nm" means